src/main/rdf/vocab/unit/VOCAB_QUDT-UNITS-ALL.ttl

```diff
@@ -33979,7 +33979,7 @@ unit:MilliW-PER-M2-NanoM-SR
   qudt:conversionMultiplier 1000000.0 ;
   qudt:conversionMultiplierSN 1.0E6 ;
   qudt:hasDimensionVector qkdv:A0E0L-1I0M1H0T-3D0 ;
```

Comment:
Inconsistent dimension vector with the associated QuantityKind

Reply:
Don't understand. I just added the quantity kind to that term(MilliW-PER-M2-NanoM-SR) which was previously unknown. Didn't touch any other lines.
My term unit for spectral radiance W m–2 sr–1 μm–1 seems to match it up to a constant